frontend/src/lang/modules/en.ts

```diff
@@ -1053,7 +1053,7 @@ const message = {
             hostname: 'Hostname',
             passwd: 'System password',
             passwdHelper: 'Input characters cannot include $ and &',
-            timeZone: 'System time zone',
+            timeZone: 'Time zone',
             localTime: 'Server time',
             timeZoneChangeHelper: 'Modifying the system time zone requires restarting the service. Continue?',
             timeZoneHelper: `If you don't install "timedatectl" command, you may not change the time zone. Because system uses that command to change time zone.`,
```

Comment:
The changes in this code snippet are primarily related to string translations. The most noticeable change is the replacement of "'System time zone'" with "'Time zone'". This might be an intended translation update or a typo.

There are no other specific irregularities, potential issues, or optimization suggestions based on the provided information. If further context or additional modifications are needed, they should be discussed separately to ensure thorough review and consideration.